DatadogSessionReplay/Sources/Processor/Diffing/Diff+SRWireframes.swift

```diff
@@ -41,17 +41,32 @@ extension SRWireframe: Diffable {
             return true
         }
     }
+
+    var type: String {
+        switch self {
+        case let (.shapeWireframe(value)):
+            return value.type
+        case let (.textWireframe(value)):
+            return value.type
+        case let (.imageWireframe(value)):
+            return value.type
+        case let (.placeholderWireframe(value)):
+            return value.type
+        case let (.webviewWireframe(value)):
+            return value.type
+        }
+    }
 }
 
 // MARK: - Resolving Mutations
 
 internal typealias WireframeMutation = SRIncrementalSnapshotRecord.Data.MutationData.Updates
 
-internal enum WireframeMutationError: Error {
+internal enum WireframeMutationError: Error, Equatable {
```

Comment:
I believe this enum is automatically conforming to `Equatable` since the associated values you added are `String`. I don't think it's an issue to add it explicitly, but I was wondering if I missed anything here?

Reply:
Assertions in test were complaining about it. I think that once enum has associated values it needs to be explicitly specified